test/Libraries/CoreNodesTests/ListTests.cs

```diff
@@ -950,5 +950,206 @@ public static void Chop6()
 
             Assert.Throws<InvalidCastException>(() => { List.Chop(list, lengths); });
         }
+
+        [Test]
```

Comment:
are there any tests for SortByFunction? - are these covered somewhere else in the tests? If not can we add some?